bootstrap/flags/flags_test.go

```diff
@@ -96,3 +96,17 @@ func TestNewOverrideRegistry(t *testing.T) {
 
 	assert.Equal(t, expectedRegistryUrl, actual.RegistryUrl())
 }
+
+func TestDashR(t *testing.T) {
```

Comment:
Please add a test for our most common use which is `"-cp=consul.http://edgex-core-consul:8500", "--registry", "--confdir=/res"` to make sure the first conditional for -cp changed to regex is still handling this properly.

Reply:
Note that the expected value for UseRegistry() is true and for RegistryUrl is constant `UseRegistryNoUrlValue = "."`

Reply:
Good test. Corrected an issue. Just pushed.